packages/shared/lib/core/network/interfaces/network.interface.ts

```diff
@@ -1,4 +1,4 @@
-import { TokenMetadata } from '@lib/typings/assets'
+import { ITokenMetadata } from '@core/wallet'
 import { NetworkProtocol } from '../enums/network-protocol.enum'
 import { NetworkType } from '../enums/network-type.enum'
```

Comment:
These should be in one import if they're barrel exported correctly.

Reply:
Done